client/ayon_core/hosts/substancepainter/plugins/publish/validate_ouput_maps.py

```diff
@@ -30,11 +30,16 @@ def process(self, instance):
         # it will generate without actually exporting the files. So we try to
         # generate the smallest size / fastest export as possible
         config = copy.deepcopy(config)
+        invalid_channels = self.get_invalid_channels(instance, config)
+        if invalid_channels:
+            raise PublishValidationError(
+                "No Channel(s){} found in the texture set:{}".format(
+                    invalid_channels, instance.name
+                ))
```

Comment:
Something fishy going on with this validation error. Why are formatting `invalid_channels` list halfway in the string there? :) Plus, if there are invalid channels why does it report that there are no channels found?